nebuly-platform/values.yaml

```
@@ -348,12 +346,14 @@ ingestionWorker:
     entitiesBatchSize: 20000
     # -- Batch size of interactions loaded in each step of enrich interactions.
     enrichInteractionBatchSize: 10000
+    # -- Enable language detection for PII detection.
+    piiEnableLanguageDetection: false
```

I suggest renaming this to keep the feature flags naming consistent (it makes it easier to find them)

```suggestion
    enablePiiLanguageDetection: false
```

---

done in https://github.com/nebuly-ai/helm-charts/pull/72/commits/a1854db1e2caecb8caf225f34e8c41e211b9ba87